### PR TITLE
删除FAQ和FAQ_CN中的多余符号

### PR DESCRIPTION
**Pull request by Miss-you**
### Summary

删除FAQ和FAQ_CN中的多余符号

### CODE REVIEW

File: FAQ.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ In addition, APISIX has dynamic routing and hot loading of plug-ins, which is es
 
 ## What's the performance of APISIX?
 
-One of the goals of APISIX design and development is the highest performance in the industry. Specific test data can be found here：https://github.com/iresty/apisix/blob/master/doc/benchmark-cn.md。
+One of the goals of APISIX design and development is the highest performance in the industry. Specific test data can be found here：https://github.com/iresty/apisix/blob/master/doc/benchmark-cn.md.
```
Comment (juzhiyuan) [inline FAQ.md]: It is recommended to remove the symbol here, keep the same as Chinese version.

Comment (juzhiyuan) [reply FAQ.md]: Also, we'd better use the `[Title](Link)` Markdown Hyperlink syntax here.

Comment (Miss-you) [reply FAQ.md]: fixed. I have change the Hyperlink format.

File: FAQ.md [hunk 2]
```diff
@@ -14,7 +14,7 @@ In addition, APISIX has dynamic routing and hot loading of plug-ins, which is es
 
 ## What's the performance of APISIX?
 
-One of the goals of APISIX design and development is the highest performance in the industry. Specific test data can be found here：https://github.com/iresty/apisix/blob/master/doc/benchmark-cn.md。
+One of the goals of APISIX design and development is the highest performance in the industry. Specific test data can be found here：[benchmark](https://github.com/iresty/apisix/blob/master/doc/benchmark-cn.md)
```
Comment (juzhiyuan) [inline FAQ.md]: Please use this link: https://github.com/iresty/apisix/blob/master/doc/benchmark.md

**Pull request merged**